### ISSUE TITLE
Cannot find namespace 'svgson'.

### ISSUE THREAD

**Issue by PhantomRay** [DiceBear/avatars#50]
Trying to use this library but have a compilation error.

```
node_modules/@dicebear/avatars/lib/index.d.ts:12:90 - error TS2503: Cannot find namespace 'svgson'.

Line 12: export declare type SpriteCollection<O = {}> = (random: Random, options?: O) => string | svgson.schema;
```

Any idea to fix this?

**Comment by eshengsky** [comment]
I resolved it with following:
in `tsconfig.json` fille

```
"types": ["@dicebear/avatars/typings/svgson"]
```

**Comment by PhantomRay** [comment]
Thank you.

**Comment by FlorianKoerner** [comment]
Should of course also work without additional configuration. I'll have a look at it.

**Comment by Orokon** [comment]
I have the same error but for me, adding the line to types is not resolving it.
I am using it like this:

`import Avatars from '@dicebear/avatars';`
`import initials from '@dicebear/avatars-initials-sprites';`

`const avatars = new Avatars(initials, { base64: true, width: 40, height: 40, margin: 5, backgroundColors: ['yellow'], bold: true });`

And in some other place I am using it the same way but with jdenticons.

**Comment by Orokon** [comment]
I could get it to work with adding the typings folder to typeRoots and using also "types" but that means I have to specify all modules that I'm using in types like node or jasmine.
We still need a way without additional configuration.

**Comment by elierotenberg** [comment]
Same problem here. I usually have a `shim.d.ts` around for stuff like that, I ended up adding `import "@dicebear/avatars/typings/svgson";` to it instead of changing my `tsconfig.json`.

**Comment by FlorianKoerner** [comment]
Should be fixed with version 4.0.3. Thanks for your patience.
